amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/activity/InterchangeEndpoints.java

```diff
@@ -420,6 +420,18 @@ public JsonBean getProject(@PathParam("projectId") Long projectId, JsonBean filt
         return InterchangeUtils.getActivity(projectId, filter);
     }
 
+    @GET
+    @Path("/projects/info/{projectId}")
```

Comment:
If you don't mind, let's make the path a bit simpler & clearer `/info/{projectId}`

Reply:
done